makefile

```diff
@@ -15,4 +15,7 @@ console:
 	env $$(cat .env) irb -r ./app
 
 db:
-	env $$(cat .env) ruby create_table.rb
+	env $$(cat .env) ruby ./db/create_table.rb
+
+test:
```

Comment:
Yes, there must be spaces instead of tabs somewhere. What I would do is duplicate the db command as it is and then replace the texts (that's the little trick I do :-) )